test/unit/mock-tooling.ts

```diff
@@ -19,4 +30,26 @@ describe('mock-tooling', () => {
       return expect(() => mockTooling.mockTokeninfoEndpoint({ url: ''} as MockOptions)).to.throw(Error);
     });
   });
+
+  describe('mockAccessTokenEndpointWithErrorResponse', () => {
```

Comment:
semanticially something like `mocked AccessTokenEndpointWithErrorResponse`makes more sense since we are not testing the returned value of 'mockAccessTokenEndpointWithErrorResponse' but the behavior of the installed mock.

Reply:
...called by the function mockAccessTokenEndpointWithErrorResponse - it is the test of the function